Mage.Server/src/main/java/mage/server/MageServerImpl.java

```diff
@@ -1071,28 +1071,26 @@ public void disconnectUser(final String sessionId, final String userSessionId) t
     @Override
     public void muteUser(final String sessionId, final String userName, final long durationMinutes) throws MageException {
         execute("muteUser", sessionId, () -> {
-            User user = UserManager.instance.getUserByName(userName);
-            if (user != null) {
```

Comment:
same idea here as above - perhaps I don't understand the implications of changing to Java optionals in terms of null ref exceptions when not checking on isPresent